include/plat/pc99/plat/machine/devices.h

```diff
@@ -10,7 +10,13 @@
 
 #define PPTR_APIC KDEV_BASE
 
+#if defined(CONFIG_VGA_PRINTING)
+#define PPTR_VGA    (PPTR_APIC + BIT(PAGE_BITS))
+#define PPTR_IOAPIC_START (PPTR_VGA + BIT(PAGE_BITS))
+#else
```

Comment:
I wonder, so you want to make this exclusive or should both output options be available at the same time also for convenience?

Reply:
Good question. I presumed if the COM port was available it'd be more convenient to use that but I could see there being some value in having both. I can rework it to allow both devices to be used if you'd like

Reply:
I started working on this but realized if this were the case we'd have some repeat functions like `kernel_putDebugChar` and `serial_init`. There is probably a workaround for this but it could make things more confusing/entangled do we really want too make sure that a user can have both on at the same time?

Reply:
I think it'd be fine to leave that for a future iteration.